### DISCUSSION TITLE
歡迎來到伴伴學討論區！

### DISCUSSION THREAD

**Discussion by accomdemy** [accomdemy/accomdemy#11] [Announcements]
## 👋 歡迎!
  我們把討論區作爲我們社區成員相互聯係的地方。在這裏，我們希望您可以：
  * 提出您的問題
  * 分享您的想法
  * 與其他社區成員互動
  * 歡迎他人，保持開放的心態。
  請記住，這是一個大家共同努力建立的社群 💪.

作爲開始，請在下方評論區自我介紹，并且告訴大家你爲什麽來到這個社群。

**Comment by S10143806H** [comment]
大家好，我是朱琪，大家也可以叫我SUSHI，來自新加坡的瑞昱新手工程師，也是伴伴學的社工。
希望能在伴伴學和大家一起共同進步！
